### PR TITLE
set home server from #!ctl

### PR DESCRIPTION
**Pull request by benharri**
initial draft of setting homeserver from #!ctl. not yet tested.

### CODE REVIEW

**Review by fosslinux — APPROVED** [review]

File: bin/hashbangctl [hunk 1]
```diff
@@ -46,6 +46,7 @@ user = {
 menu_items = [
     ('n',"Change Name"),
     ('s',"Change Shell"),
+    ('h',"Change Homeserver"),
```
Comment (RyanSquared) [inline bin/hashbangctl]: taken by help

**Review by fosslinux — COMMENTED** [review]
ignore me

File: bin/hashbangctl [hunk 2]
```diff
@@ -141,6 +143,23 @@ while True:
...
         print('\nPlease press [Enter] to continue')
 
+    elif key == 'h':
+        hosts = [
+            "da1.hashbang.sh",
+            "ny1.hashbang.sh",
+            "to1.hashbang.sh",
+            "sf1.hashbang.sh"
+        ]
+        host = raw_input("Enter new homeserver, eg {da,ny,to,sf}1.hashbang.sh")
```
Comment (RyanSquared) [inline bin/hashbangctl:147]: either make this python2/3 compatible or change the hashbang line to specify python version

File: bin/hashbangctl [hunk 3]
```diff
@@ -141,6 +143,23 @@ while True:
...
+
+        if not host or not host.endswith(".hashbang.sh") or host not in hosts:
```
Comment (RyanSquared) [inline bin/hashbangctl]: should only need to check `if host not in hosts`

**Review by RyanSquared — CHANGES_REQUESTED** [review]

File: bin/hashbangctl (hunk 1 above)
Comment (benharri) [reply bin/hashbangctl]: resolved

File: bin/hashbangctl (hunk 2 above)
Comment (benharri) [reply bin/hashbangctl:147]: that probably belongs in a separate PR

File: bin/hashbangctl [hunk 4]
```diff
@@ -141,6 +143,23 @@ while True:
 
         print('\nPlease press [Enter] to continue')
 
+    elif key == 'H':
+        hosts = [
```
Comment (daurnimator) [inline bin/hashbangctl]: Could you fetch this list from the database?

**Review by daurnimator — CHANGES_REQUESTED** [review]

File: bin/hashbangctl (hunk 4 above)
Comment (benharri) [reply bin/hashbangctl]: should be resolved

**Comment by lrvick** [comment]
Sorry this never merged, but in light of the go-daemon rewrite this won't be needed now.

Your help on the new go daemon is very much welcome though!

**Pull request closed without merging**